proto/uber/cadence/api/v1/service_worker.proto

```diff
@@ -169,6 +169,7 @@ message PollForDecisionTaskResponse {
   google.protobuf.Timestamp started_time = 13;
   map<string, WorkflowQuery> queries = 14;
   int64 next_event_id = 15;
+  int64 history_size = 16;
```

Comment:
from chat: since this is the cumulative number of bytes in the workflow, `total_history_bytes` or something seems a lot more descriptive

Reply:
renaming finished